src/manage_nsfs/manage_nsfs_cli_responses.js

```diff
@@ -94,25 +105,30 @@ ManageCLIResponse.AccountList = Object.freeze({
 
 ManageCLIResponse.BucketCreated = Object.freeze({
     code: 'BucketCreated',
+    message: 'Bucket has been successfully created',
```

Comment:
I would put the 'successfully' in the end of the sentance

Reply:
I think I should do this for all the operations. Thanks